packs/opentelemetry_collector/templates/opentelemetry_collector.nomad.tpl

```diff
@@ -45,12 +51,15 @@ job [[ template "full_job_name" . ]] {
 
         ports = [[ keys $vars.network_config.ports | toPrettyJson ]]
 
+        [[ if $vars.use_volumes ]]
```

Comment:
In which situation would this be set to `false`? 

Without the `volumes` config, the Otel config would not be mounted into the container and I think that, in general, users would always want to use their own config, but I may be missing something 😅

Reply:
I've actually never mounted a volume for the Otel collector config. Just always mounted to the container filesystem and it still picks up the user-defined one in the examples file without any issues.

Reply:
Ah ok. I think I got a bit confused with the naming. Stateful storage usually makes me think of something like CSI or host volumes, which require [`volume`](https://www.nomadproject.io/docs/job-specification/volume) config block.

But in this case, it's mounting the configuration file from the alloc dir into the container, which is required to allow the container to access the file.

So that's why I think we could do without this variable, these volumes always have to be there.

Reply:
Gotcha. I've removed the flag and re-tested.